SharedPackages/BrowserServicesKit/Sources/PixelExperimentKitTestUtils/MockInternalUserDecider.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  MockInternalUserDecider.swift
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import BrowserServicesKit
+import Combine
+import Foundation
+import PixelExperimentKit
+
+public final class MockInternalUserDecider: InternalUserDecider {
```

Comment:
Please refer to this comment for more information on this change: https://github.com/duckduckgo/apple-browsers/pull/562/files#r2055883698

I needed to make these mocks available to the main app.

Reply:
Removed.